rel-eng/push-packages-to-obs.sh

```diff
@@ -253,9 +253,11 @@ while read PKG_NAME; do
   # check which endpoint we are using to match the product
   if [ "${OSCAPI}" == "https://api.suse.de" ]; then
       PRODUCT_VERSION=$(sed -n 's/.*web.version\s*=\s*\(.*\)$/\1/p' ${BASE_DIR}/../web/conf/rhn_web.conf)
+      SEMANTIC_VERSION=$(echo ${PRODUCT_VERSION,,} | sed 's/\s.*$//')
   else
       # Uyuni settings
       PRODUCT_VERSION=$(sed -n 's/.*web.version.uyuni\s*=\s*\(.*\)$/\1/p' ${BASE_DIR}/../web/conf/rhn_web.conf)
+      SEMANTIC_VERSION=${PRODUCT_VERSION}.0
```

Comment:
If you wanted to remove the leading zero for the month, this does not work:
```
$ BASE_DIR=${PWD}
$  RODUCT_VERSION=$(sed -n 's/.*web.version.uyuni\s*=\s*\(.*\)$/\1/p' ${BASE_DIR}/../web/conf/rhn_web.conf)
$  EMANTIC_VERSION=${PRODUCT_VERSION}.0
$ echo $SEMANTIC_VERSION
2022.07.0
```
So I am not sure what you mean with the leading 0

Reply:
Same, that was the old code. should be better now after pushing the changes